### PR TITLE
Feature/549 add rsvp badge to event lists

### PR DESCRIPTION
**Pull request by steventlamb**

### CODE REVIEW

File: src/nyc_trees/apps/home/tests.py [hunk 1]
```diff
@@ -31,7 +31,9 @@ def requery_user(self):
     def test_visit_marks_boolean_true(self):
         """assert that any successful view function, when wrapped with
         the decorator, will mark the associated boolean"""
-        view = lambda request: {}
+        def pedantic_flake_8_view(request):
```
Comment (jwalgran) [inline src/nyc_trees/apps/home/tests.py]: Do we need the assignment or can just:

```python
def view(request):
    return {}
```

Comment (steventlamb) [inline src/nyc_trees/apps/home/tests.py]: we do not need the assignment, I did this to document why this was happening in as few characters as possible

**Comment by steventlamb** [comment]
@jwalgran followed your suggestion on the lintfixer

**Comment by jwalgran** [comment]
+1 the +RSVP label now shows up on all the event lists.

**Pull request merged**